src/wagtailbakery/api_views.py

```diff
@@ -85,7 +89,7 @@ def build_object(self, obj):
         self.build_file(path, page_content)
 
     def build_queryset(self):
-        [self.build_object(o) for o in self.get_queryset().all()]
+        [self.build_object(o) for o in self.get_queryset().all() if o.is_root() is False]
```

Comment:
you could do 
```python
[self.build_object(o) for o in self.get_queryset().filter(depth__gt=1)]
``` as only root has depth 1.

or, at least 

```python
[self.build_object(o) for o in self.get_queryset().all() if not o.is_root()]
```

Reply:
query filter would be way more efficient will update